server/pulp/server/webservices/views/repositories.py

```diff
@@ -0,0 +1,1214 @@
+        ret['_href'] = reverse('repo_publish_schedule_resource', kwargs={
+            'repo_id': repo_id, 'distributor_id': distributor_id, 'schedule_id': schedule.id
+        })
+        response = generate_json_response_with_pulp_encoder(ret)
+        return generate_redirect_response(response, ret['_href'])
+
+
+class RepoPublishScheduleResourceView(ScheduleResource):
+    """
+    View for a single scheduled publish.
+    """
+
+    def __init__(self):
+        """
+        Initialize RepoPublishScheduleResourceView.
+        """
+
+        super(RepoPublishScheduleResourceView, self).__init__()
+        self.manager = manager_factory.repo_publish_schedule_manager()
+
+    @auth_required(authorization.READ)
+    def get(self, request, repo_id, distributor_id, schedule_id):
+        """
+        Retrieve information about a scheduled publish.
+
+        :param request: WSGI request object
+        :type  request: django.core.handlers.wsgi.WSGIRequest
+        :param repo_id: id of the repository
+        :type  repo_id: str
+        :param distributor_id: id of the distributor
+        :type  distributor_id: str
+        :param schedule_id: id of the scheduled publish to be retrieved
+        :type  schedule_id: str
+
+        :return: Response containing serialized dict for the scheduled publish
+        :rtype : django.http.HttpResponse
+        """
+
+        self.manager.validate_distributor(repo_id, distributor_id)
+        resource_href = reverse('repo_publish_schedule_resource', kwargs={
+            'repo_id': repo_id, 'distributor_id': distributor_id, 'schedule_id': schedule_id
+        })
+        return self._get(schedule_id, resource_href)
+
+    @auth_required(authorization.UPDATE)
+    @json_body_required
+    def put(self, request, repo_id, distributor_id, schedule_id):
+        """
+        Update a scheduled publish.
+
+        :param request: WSGI request object
+        :type  request: django.core.handlers.wsgi.WSGIRequest
+        :param repo_id: id of the repository
+        :type  repo_id: str
+        :param distributor_id: id of the distributor
+        :type  distributor_id: str
+        :param schedule_id: id of the scheduled publish to be updated
+        :type  schedule_id: str
+
+        :return: Response containing serialized dict for the scheduled publish
+        :rtype : django.http.HttpResponse
+        """
+
+        updates = request.body_as_json
+        if 'schedule' in updates:
+            updates['iso_schedule'] = updates.pop('schedule')
+        schedule = self.manager.update(repo_id, distributor_id, schedule_id, updates)
+        ret = schedule.for_display()
+        ret['_href'] = reverse('repo_publish_schedule_resource', kwargs={
+            'repo_id': repo_id, 'distributor_id': distributor_id, 'schedule_id': schedule_id
+        })
+        return generate_json_response(ret)
+
+    @auth_required(authorization.DELETE)
+    def delete(self, request, repo_id, distributor_id, schedule_id):
```

Comment:
would you implement same try/except like in scheduled sync removal? https://github.com/pulp/pulp/pull/1742/files#diff-69de4359dc90d2ab726dbca11e4f32b4R565